src/ActionModal/components/ActionModal.vue

```diff
@@ -790,45 +800,39 @@ export default {
           }
         }
       `,
+      /* istanbul ignore next */
       variables() {
-        /* istanbul ignore next */
         return {
           networkId: this.networkId
         }
       },
+      /* istanbul ignore next */
       update(data) {
-        /* istanbul ignore next */
-
         return data.network
       }
     },
     $subscribe: {
       userTransactionAdded: {
+        /* istanbul ignore next */
         variables() {
-          /* istanbul ignore next */
           return {
             networkId: this.networkId,
             address: this.session.address
           }
         },
+        /* istanbul ignore next */
         skip() {
-          /* istanbul ignore next */
           return !this.txHash
         },
         query: UserTransactionAdded,
+        /* istanbul ignore next */
         result({ data }) {
```

Comment:
I actually think we should test this as it is more complex. but maybe for another time